app/components/ui/InfoBar.vue

```
@@ -13,7 +13,11 @@ const disablePath = computed(() => {
     v-if="!disablePath"
     class="z-10 flex flex-col items-center justify-between bg-transparent py-4 text-sm text-inherit font-normal md:(flex-row px-9 py-2 space-y-0) sm:(flex-row px-6 py-2) space-y-2 2xl:px-43 xl:px-30"
   >
-    <UiWeather />
+    <!-- Breadcrumb -->
+    <UiBreadcrumbs />
+    
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)